AudioPlayerRecorderSamples/build.gradle

```diff
@@ -24,7 +24,10 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:19.+'
+    // After update, check if activity still being recreated after permission granted,
+    // if no - remove RecorderActivity.STATE_START_RECORD
```

Comment:
Also this comment needs updating. It should probably be `RecorderActivity#mStartRecord`.